PallyPower.lua

```
@@ -1618,6 +1619,78 @@ function PallyPower:CHAT_MSG_SYSTEM(event, text)
+                    for pally in pairs(AllPallys) do
+
+                        if PallyPower_Assignments[pally][PallyPower.ClassToID[englishClass]] then
+                            -- self:Print(PallyPower_Assignments[pally][PallyPower.ClassToID[englishClass]])
+                            -- self:Print(GetNormalBlessings(pally, PallyPower.ClassToID[englishClass], playerName))
+                            normalBlessID = GetNormalBlessings(pally, PallyPower.ClassToID[englishClass], playerName)
+                            -- self:Print(normalBlessID)
+                            if normalBlessID then
+                                -- self:Print(normalBlessID)
+                                normalBlessText = " (single bless: " .. PallyPower.IDToBless[tonumber(normalBlessID)] .. ") "
+                            else
+                                normalBlessText = ""
+                            end
+
+                            SendChatMessage(pally .. " -> " .. PallyPower.IDToBless[PallyPower_Assignments[pally][PallyPower.ClassToID[englishClass]]] .. normalBlessText, "WHISPER", "Common", playerName)
+                        end
+
+                    end
+                    -- self:Print(PallyPower_Assignments[PallyPower.player][PallyPower.ClassToID[englishClass]])
+                    return false
+                end
+                -- self:Print(text)
+                -- self:Print(PallyPower.player)
+                -- self:Print(englishClass)
+                -- self:Print(PallyPower.ClassToID[englishClass])
+
+                SetNormalBlessings(PallyPower.player, PallyPower.ClassToID[englishClass], playerName, blessID)
+
+                self:Print(playerName .. " has now assigned " .. blessName .. " bless (small)")
+                SendChatMessage("You have now assigned " .. blessName .. " bless (small)", "WHISPER", "Common", playerName)
+            else
+                self:Print("Player not found. Is he/she in your group?")
```

if a player is shown this message, what are they supposed to do about it?

either silently discard or error reply to the sender, don't show anything to user